charts/operator/templates/clusterrole-operator.yaml

```diff
@@ -18,13 +18,9 @@ rules:
   - apiGroups: ["scheduling.azimuth.stackhpc.com"]
     resources: ["*"]
     verbs: ["*"]
-  # allow these things to be deleted by the operator
-  - apiGroups: ["caas.azimuth.stackhpc.com"]
-    resources: ["clusters"]
-    verbs: ["*"]
-  - apiGroups: [""]
-    resources: ["configmaps"]
-    verbs: ["*"]
-  - apiGroups: ["rbac.authorization.k8s.io"]
-    resources: ["clusterrolebindings"]
-    verbs: ["get", "list", "create", "update", "patch"]
+  # The operator has fetch and delete permissions for every resource
+  - apiGroups: ["*"]
```

Comment:
I would much rather this was more specific. It feels like an honey pot letting it access everything.

Maybe we need to add these permissions higher up our tree of operators somewhere?

For now, could we just add the extra bits we need.